docs/examples/drag-and-drop/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I've been getting rid of the html,head, and body tags since they're optional in html5. Maybe just a meta tag with the lang would be sufficient?

Reply:
Removed the optional tags